R/plots.R

```diff
@@ -270,19 +273,22 @@ plotLoadResidualsCM <- function(...){
 #'   
 #' @param load.model use this to pull names of date column and constituent name
 #' @param observations these are the observations to plot the residuals for
+#' @param residuals the observation residuals 
 #' @param dateField name of the date field in observations
 #' @param verbose debugging flag
 #' @param day.of.year flag if true will plot residuals by the day of year to,
 #'   can help user look for seasonality
-plotResidualsCM <- function(type, load.model, observations, dateField="Date", verbose=FALSE,  xObservations=FALSE, xFlow=FALSE, day.of.year=FALSE){
- 
+plotResidualsCM <- function(type, load.model, observations,resids, dateField="Date", verbose=FALSE,  xObservations=FALSE, xFlow=FALSE, day.of.year=FALSE){
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes_string=geom_point=yday=xlab=NULL
```

Comment:
pretty sure resids doesn't need to be passed in. but again, plots will be rewritten.